rust/xaynet-mobile/src/ffi/mod.rs

```diff
@@ -6,6 +6,7 @@ pub use participant::*;
 mod settings;
 pub use settings::*;
 
+pub use ffi_support::{ByteBuffer, FfiStr};
```

Comment:
Why do we need `FfiStr`? I don't see it being used in the diff

Reply:
it is used in the `participant` and `settings` mods, which are exposed as part of the publich api.